bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -233,6 +239,22 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
         return allAssetFolders;
     }
 
+    private String applyUrlRewrites(String url) {
+        for (String rewrite : urlRewrites) {
```

Comment:
it might be more efficient (or atleast clear) if, in activate(..) you parse the params out into a Map using [1]

And in this you just check the parsed map values?

[1] https://github.com/Adobe-Consulting-Services/acs-aem-commons/blob/master/bundle/src/main/java/com/adobe/acs/commons/util/ParameterUtil.java#L102

Reply:
Addressed in the latest commit